### PR TITLE
Create fixschools, fixmajors commands. Improved search.

### PR DESCRIPTION
**Pull request by theCreedo**

### CODE REVIEW

File: scripts/rename_schools.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+				non_edus.append(user)
+
+		for key, value in edu_groups.iteritems():
+			count_schools_dict = {}
+			for user in value:
+				if not user.school_name in count_schools_dict.keys():
+					count_schools_dict[user.school_name] = 1
+				else:
+					count_schools_dict[user.school_name]+=1
+			# TODO: Sort in order and get top 5
```
Comment (rjchee) [inline scripts/rename_schools.py]: Is this TODO resolved?

File: scripts/rename_schools.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		edu_groups = {}
+		non_edus = []
+
+		for user in all_users:
+			if user.type == 'admin':
+				continue;
+			if "edu" in user.email:
+				school_edu = user.email.split("@")
+				if not school_edu[1] in edu_groups.keys():
+					edu_groups[school_edu[1]] = [user]
```
Comment (rjchee) [inline scripts/rename_schools.py]: if you use a defaultdict, you don't need this case

File: scripts/rename_schools.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+					count_schools_dict[user.school_name]+=1
+			# TODO: Sort in order and get top 5
+			sorted_schools_dict = sorted(count_schools_dict, key=count_schools_dict.get, reverse=True)[:5]
+			print(key)
+			# for x in sorted_schools_dict:
+			# 	print(x);
+			temp = 0
+
+			for x in sorted_schools_dict:
+				if not x == None:
```
Comment (rjchee) [inline scripts/rename_schools.py]: `x is not None`

File: scripts/rename_schools.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+				else:
+					edu_groups[school_edu[1]].append(user)
+			else: 
+				non_edus.append(user)
+
+		for key, value in edu_groups.iteritems():
+			count_schools_dict = {}
+			for user in value:
+				if not user.school_name in count_schools_dict.keys():
+					count_schools_dict[user.school_name] = 1
```
Comment (rjchee) [inline scripts/rename_schools.py]: This can be done without two cases with a `collections.Counter`

File: scripts/rename_schools.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+						break
+					elif is_decide == 'N' or is_decide == 'n':
+						continue
+					else:
+						print('Invalid decision - Try again.')
+					print('\n')
+			for user in value:
+				user.school_name = decided_name
+				DB.session.add(user);
+				DB.session.commit();
```
Comment (rjchee) [inline scripts/rename_schools.py]: You could commit outside of the loop to make it faster

File: scripts/rename_schools.py [hunk 6]
```diff
@@ -0,0 +1,89 @@
+from pepper.users.models import User
+from flask_script import Command
+import operator
+from pepper.app import DB
+
+class FixUsersSchoolNames(Command):
+    def run(self):
+		all_colleges = []
```
Comment (rjchee) [inline scripts/rename_schools.py]: Shouldn't you be memorizing mappings from misspellings to actual college names?

File: scripts/rename_schools.py (hunk 1 above)
Comment (theCreedo) [reply scripts/rename_schools.py]: Yes it is

File: scripts/rename_schools.py (hunk 2 above)
Comment (theCreedo) [reply scripts/rename_schools.py]: Resolved

File: scripts/rename_schools.py (hunk 3 above)
Comment (theCreedo) [reply scripts/rename_schools.py]: Resolved

File: scripts/rename_schools.py (hunk 5 above)
Comment (theCreedo) [reply scripts/rename_schools.py]: Resolved

**Pull request merged**